content/hi/docs/setup/best-practices/multiple-zones.md

```diff
@@ -0,0 +1,124 @@
+---
+reviewers:
+
+
+
+title: कई क्षेत्रों में चलाना
+weight: 20
+content_type: concept
+---
+
+<!-- overview -->
+
+यह पृष्ठ कई क्षेत्रों में कुबेरनेट्स चलाने का वर्णन करता है।
```

Comment:
Can you use more simpler wording for this `पृष्ठ कई क्षेत्रों` part?

Reply:
which one should i use 
- यह `पेज` कई `ज़ोन` में कुबेरनेट्स चलाने का वर्णन करती है।
- यह पृष्ठ कई `ज़ोन` में कुबेरनेट्स चलाने का वर्णन करती है।
- यह `पेज` कई क्षेत्रों में कुबेरनेट्स चलाने का वर्णन करती है।

The english sentence was `This page describes running Kubernetes across multiple zones.`